game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTransportUtils.java

```diff
@@ -244,12 +243,12 @@ public static int findUnitsTransportCost(final List<Unit> units) {
     results.add(unit);
     if (Matches.unitIsLandTransportWithoutCapacity().test(unit)) {
       units.sort(
-          Comparator.<Unit>comparingInt(u -> TripleAUnit.get(u).getMovementLeft().intValue())
+          Comparator.<Unit>comparingInt(u -> u.getMovementLeft().intValue())
               .thenComparing(getDecreasingAttackComparator(player)));
       results.add(units.get(0));
     } else {
       units.sort(
-          Comparator.<Unit>comparingInt(u -> TripleAUnit.get(u).getMovementLeft().intValue())
+          Comparator.<Unit>comparingInt(u -> u.getMovementLeft().intValue())
```

Comment:
@ron-murhammer You might have any insight here.
Is it important that movement left is rounded to an  int before comparing units?

Otherwise it would be simpler writing something like
```
Comparator.comparing(Unit::getMovementLeft)
```